stdlib/transactions/src/main/ballerina/src/transactions/2pc_participant.bal

```diff
@@ -15,6 +15,7 @@
 // under the License.
 
 import ballerina/log;
+import ballerina/io;
```

Comment:
Shall we order the imports alphabetically?

Reply:
fixed